<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUD Cliente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 52, 52);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>351</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(127, 127, 127);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>Torna Indietro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_crea_cliente">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>Crea Cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_modifica_cliente">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>üñäÔ∏è</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../lory1/Downloads/matita (72px).ico</normaloff>../../../../../../lory1/Downloads/matita (72px).ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bottone_elimina_cliente">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20; color:white;background-color:rgb(38, 157, 206)
</string>
   </property>
   <property name="text">
    <string>üóëÔ∏è</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../../lory1/Downloads/cestino (72px).ico</normaloff>../../../../../../lory1/Downloads/cestino (72px).ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Clienti:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
